nikola/plugins/compile/ipynb/__init__.py

```diff
@@ -32,7 +32,15 @@
 
 try:
     from IPython.nbconvert.exporters import HTMLExporter
-    from IPython.nbformat import current as nbformat
+    import IPython.nbformat
+    if IPython.version_info[0] >= 3:     # API changed with 3.0.0
+        def nbformat_reads(s):
+            # Update to the current format so that the export works.
+            return IPython.nbformat.reads(
+                s, IPython.nbformat.current_nbformat)
+    else:
+        nbformat_reads = IPython.nbformat.current.reads_json
```

Comment:
Also, you would have to `import IPython.nbformat.current` and work with it all the time.

Here is a revised solution: